docs/home/SECURITY.md

```diff
@@ -147,23 +151,128 @@ The list does not include ports that are bound to the loopback interface (localh
 
     - 8200 - REST API
 
+### Connection protocols and ciphers used by components in Epiphany
+
+Below you can find list of cipersuites and protocols used for communication set in Epiphany on per component basis. This
+doesn't contain information about deprecated components: Apache Ignite, Istio, Hashicorp Vault.
+
+1. OS services:
+
+    - SSH
+        - ciphersuites:  
+          chacha20-poly1305@openssh.com,aes256-gcm@openssh.com,aes128-gcm@openssh.com,aes256-ctr,aes192-ctr,aes128-ctr
+        - macs:  
+          hmac-sha2-512-etm@openssh.com,hmac-sha2-256-etm@openssh.com,umac-128-etm@openssh.com,hmac-sha2-512,hmac-sha2-256,umac-128@openssh.com
+        - kexalgorithms:  
+          curve25519-sha256@libssh.org,ecdh-sha2-nistp521,ecdh-sha2-nistp384,ecdh-sha2-nistp256,diffie-hellman-group-exchange-sha256
+
+2. HAProxy:
+
+    - protocols:  
+      TLSv1.2
+
+    - ciphersuites:  
+      ECDHE-ECDSA-AES128-GCM-SHA256:ECDHE-RSA-AES128-GCM-SHA256:ECDHE-ECDSA-AES256-GCM-SHA384:
+      ECDHE-RSA-AES256-GCM-SHA384:ECDHE-ECDSA-CHACHA20-POLY1305:ECDHE-RSA-CHACHA20-POLY1305:DHE-RSA-AES128-GCM-SHA256:
+      DHE-RSA-AES256-GCM-SHA384
+
+3. Kafka:
+
+    - protocols:  
+      TLSv1.2,TLSv1.1,TLSv1
+
+    - ciphersuites:  
+      Depends on version of Java and for Java versions older than 8u161 on JCE policy file. From update 8u161 JCE policy
+      file is not needed to enable restricted ciphers as all ciphers are enabled by default. Documentation about ciphers
+      is available under [link](https://docs.oracle.com/javase/8/docs/technotes/guides/security/SunProviders.html).
+
+4. Open Distro:
```

Comment:
Hmm, could you verify if it supports tlsv1.3? Not sure what OpenSSL version is required/privded.
https://github.com/opendistro-for-elasticsearch/security/blob/86d6292d7b2649363437e558ce3fa71c1a3b2afc/src/main/java/com/amazon/opendistroforelasticsearch/security/ssl/DefaultOpenDistroSecurityKeyStore.java#L751

Reply:
In our configuration it doesn't support. And here we are mentioning only protocols setup by Epiphany.